### PR TITLE
Adds test about mime when stream depth is reached

### PR DESCRIPTION
**Pull request by catenacyber**
## Ticket

Redmine ticket: https://redmine.openinfosecfoundation.org/issues/6367

Suricata PR incoming

### CODE REVIEW

File: tests/mime/mime-stream-depth/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Description
+
+Test mime base64 decoding completeness when stream depth is reached
```
Comment (inashivb) [inline tests/mime/mime-stream-depth/README.md:3]: Is this on the standard stream depth setting for now? 64 mb?

While I see that we're not trying to store any files, would filestore's stream depth settings also affect this (as it should ideally override stream's depth settings)?

So, should we have a suricata.yaml in this test? Or would we never change the standard settings for depths? wdyt?

Comment (catenacyber) [reply tests/mime/mime-stream-depth/README.md:3]: Yes, it is for standard depth which is 1Mb

> would filestore's stream depth settings also affect this (as it should ideally override stream's depth settings)?

Maybe

> So, should we have a suricata.yaml in this test? 

Yes, or `--set stream.depth=1mb`

**Comment by catenacyber** [comment]
Continued in https://github.com/OISF/suricata-verify/pull/1393

File: tests/mime/mime-stream-depth/README.md (hunk 1 above)
Comment (catenacyber) [reply tests/mime/mime-stream-depth/README.md:3]: `stream.reassembly.depth` in facts

**Pull request closed without merging**